alerting/migration/1738249210356-create-latest-aa-storm-reports-table.ts

```diff
@@ -0,0 +1,17 @@
+import { MigrationInterface, QueryRunner } from "typeorm";
+
+export class CreateLatestAaStormReportsTable1738249210356
+  implements MigrationInterface
+{
+  public async up(queryRunner: QueryRunner): Promise<void> {
```

Comment:
Let's do something a bit more future-proof here no?
I would add at least:
- emails (a list of emails)
- storm_name
- report_date
- activation_status
- last_triggered_at
- last_ran_at